deployment/terraform/firewall.tf

```diff
@@ -425,3 +435,13 @@ resource "aws_security_group_rule" "opensearch_ingress" {
   security_group_id        = aws_security_group.opensearch.id
   source_security_group_id = aws_security_group.app_logstash.id
 }
+
+resource "aws_security_group_rule" "app_opensearch_ingress" {
```

Comment:
What problem this ingress rule solved? Afair, there were no problems with incoming traffic to the OpenSearch previously in https://github.com/opensupplyhub/open-supply-hub/pull/220.

Reply:
It is necessary to enable a network connection from Django to OpenSearch. Previously, ingress traffic from Django to OpenSearch was blocked.